src/PowerShellEditorServices/Utility/Utils.cs

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#if CoreCLR
+using System.Runtime.InteropServices;
+#endif
+
+namespace Microsoft.PowerShell.EditorServices
+{
+    /// <summary>
+    /// General purpose common utilities to prevent reimplementation.
+    /// </summary>
+    internal static class Utils
+    {
+        /// <summary>
+        /// True if we are running on .NET Core, false otherwise.
+        /// </summary>
+#if CoreCLR
+        public static bool IsNetCore { get; } = RuntimeInformation.FrameworkDescription.StartsWith(".NET Core");
```

Comment:
Could this just be

```suggestion
        public static bool IsNetCore { get; } = true;
```

Or do we define `CoreCLR` in circumstances other than actual .net core?